docs/navigating-backup.md

```diff
@@ -41,7 +41,16 @@ The credentials/secrets management is further described within the [nautbot-plug
 repo. For the simplist use case you can set environment variables for `NAPALM_USERNAME`, `NAPALM_PASSWORD`, and `DEVICE_SECRET`. For more
 complicated use cases, please refer to the plugin documentation linked above.
 
-# Config Removals
+## Starting a Backup Job
```

Comment:
Can you cover all jobs, and add to `navigating-golden.md`?

Reply:
I can work on those. This was the only part that I have gotten to complete (and only one tried so far).

Reply:
I guess I mean, to do them all in one swoop, such as 
```
To start one of the jobs manually:

1. Navigate to the Plugin Home (Plugins->Home), with Home being in the `Golden Configuration` section
2. Select _Execute_ on the upper right buttons, then select either _Backup_, _Intended_, or _Compliance_, depending on your need. 
3. Fill in the data that you wish to have backed up
4. Select _Run Job_

```